src/OidcClientSettings.ts

```diff
@@ -102,6 +102,11 @@ export interface OidcClientSettings {
     extraQueryParams?: Record<string, string | number | boolean>;
 
     extraTokenParams?: Record<string, unknown>;
+
+    /**
+     * CORS settings for credentials sent with the refresh request.
```

Comment:
Improve the documentation:
`Credentials used by fetch with the refresh request. (default: "same-origin")`

I guess its better to explicitly set `refreshTokenCredentials` to "same-origin", to avoid possible but unlikely different browser defaults...

Reply:
defaulting to same-origin in OidcClientSettingsStore